NewForce-Cohort-4/tabloid-full-stack-team-fuschia
# Cl post details

# Description
included a details view accessible by clicking on the title of the post in the all posts view.

As a reader, I would like to see the content of a Post so I can read it.

Given a user is viewing a list of Posts
When they select a post to read
Then they should be directed to a Post Detail page that shows the Post Details.

Post Details include:

Title
Header image (if exists)
Content
Publication date (MM/DD/YYYY)
Author's Display Name

## Type of change
Please delete options that are not relevant.



# Testing Instructions
-sign in to the app
-click the "Posts" link
-click the title of one of the posts
-make sure you are taken to details view and the correct data (see above) is included

# Checklist:
